webapp/pom.xml

```
@@ -45,7 +45,7 @@
     <parent>
         <groupId>fish.payara.monitoring-console</groupId>
         <artifactId>parent</artifactId>
-        <version>1.9</version>
+        <version>1.10</version>
     </parent>
         
     <artifactId>monitoring-console-webapp</artifactId>
```

Parent pom shouldn't be released - nothing has changed

```suggestion
        <version>1.9</version>
    </parent>
        
    <artifactId>monitoring-console-webapp</artifactId>
    <version>1.10</version>
```